### PR TITLE
Add --use_move mode to bench-tps

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

Move mode controlled with a global constant. 

#### Summary of Changes

Plumb in --use_move argument to bench-tps.

Fixes #

### CODE REVIEW

File: bench-tps/src/cli.rs [hunk 1]
```diff
@@ -100,6 +102,11 @@ pub fn build_args<'a, 'b>() -> App<'a, 'b> {
                 .long("sustained")
                 .help("Use sustained performance mode vs. peak mode. This overlaps the tx generation with transfers."),
         )
+        .arg(
+            Arg::with_name("use_move")
+                .long("use_move")
```
Comment (garious) [inline bench-tps/src/cli.rs]: nit: `use-move`, such that you get `--use-move`

**Comment by garious** [comment]
> fatal error: ld terminated with signal 9 [Killed]

Hmm, repeatable linker error in a coverage build after adding a bunch of new libraries earlier that day.  I bet doubling the memory allocated to Docker fixes this.

**Comment by mvines** [comment]
Docker containers in Linux get the same memory as the host system by default and we don't make any attempt to change that.  However while the 2 cuda CI machines have 54 gb ram, the rest have only 14gb and no swap.

Try forcing the coverage job onto the cuda agent queue, like this:
https://github.com/solana-labs/solana/blob/a642168369e0aac4f499d69ecd806dffc5650648/ci/buildkite.yml#L13-L14

If that works then we just need to give the rest of the CI machines more ram (note to self: upsizing to a D8s_v3 instance looks good, that'll be 32GB RAM, similar IOPS and a modest increase in cost)

**Comment by sakridge** [comment]
@mvines cuda queue seems to have made the difference.

**Comment by mvines** [comment]
Nice.  You can land that change for now.  When we resize all the cpu VMs it can be reverted

**Pull request merged**